### ISSUE TITLE
[1Password] Passwords not displaying

### ISSUE THREAD

**Issue by natemhanson** [raycast/extensions#12005]
### Extension

https://www.raycast.com/khasbilegt/1password

### Description

 I have added the extension and chosen 1Password 7, but none of my items show up when I run the "My Passwords" command.

### Steps To Reproduce

Install 1Password extension.
Selected 1Password 7 from settings
Run the command "My Passwords"
Nothing displays

**Comment by xavibenjamin** [comment]
I'm experiencing this issue as well

**Comment by j-peeters** [comment]
Yes me too. Would love to be able to use this extension again! Thanks for looking into this.

**Comment by Luk4h** [comment]
Hello guys, Did you guys install the 1password CLI?

**Comment by j-peeters** [comment]
> Hello guys, Did you guys install the 1password CLI?

Yes I did:
![20240430 jpeeters — zsh — zsh (qterm) ▸ zsh — 80×24](https://github.com/raycast/extensions/assets/4502360/a3ccc98b-fdef-402a-ae64-1c9b29cf480b)

It just doesn't seem to be recognized by the extension.

**Comment by Luk4h** [comment]
Could you please check where it is installed with the "where op" and verify if the path is correctly configured on the extension settings

**Comment by Luk4h** [comment]
Glad I could help!
@raycastbot close this issue
